Objects/EventItem/EventManager.tscn

```diff
@@ -6,4 +6,3 @@
 script = ExtResource( 1 )
 
 [node name="Timer" type="Timer" parent="."]
-[connection signal="timeout" from="Timer" to="." method="_on_Timer_timeout"]
```

Comment:
If we remove this signal, no more items will be spawned

Reply:
I dont know how i delete this